tests/python_functional/src/syslog_ng_config/syslog_ng_config.py

```diff
@@ -132,6 +134,12 @@ def create_snmp_destination(self, **options):
     def create_db_parser(self, config, **options):
         return DBParser(config, **options)
 
+    def create_credit_card_mask(self):
+        return CreditCardMask()
+
+    def create_credit_card_hash(self):
```

Comment:
Could you please rename `create_credit_card_hash` to `create_rewrite_credit_card_hash`?

Reply:
This has been addressed by Balazs.